examples/c/src/common.cpp

```diff
@@ -88,7 +88,5 @@ std::string trim(const std::string& str) {
 }
 
 void print_usage(int /*argc*/, char** argv) {
-  std::cerr << "usage: " << argv[0] << std::endl;
-  std::cerr << "model_path = " << argv[1] << std::endl;
-  std::cerr << "execution_provider = " << argv[2] << std::endl;
+  std::cerr << "usage: " << argv[0] << " <model_path> <execution_provider>" << std::endl;
```

Comment:
What did we lose here, why is argv[1] and argv[2] not being printed anymore?

Reply:
ah, so `print_usage` here is called when user doesn't (potentially) pass model_path and/or execution provider. So the issue was that this was triggering a crash (as argv[1] / argv[2] were NULL)

Reply:
(so it's outside the scope of these `follow_config` changes.. I just threw it in here to fix the crash.

Reply:
Interesting, I pinged another teammate about it to see if they know how it was originally supposed to work.

Reply:
FYI -- In one of the commits I pushed this morning, I also added more details to the printed usage message.